action/action-dsl/src/test/java/com/powsybl/action/dsl/ActionDslLoaderTest.java

```diff
@@ -84,11 +84,11 @@ public void testFixTapDslExtension() {
         assertNotNull(fixedTapAction);
         addPhaseShifter(0);
         PhaseTapChanger phaseTapChanger = network.getTwoWindingsTransformer("NGEN_NHV1").getPhaseTapChanger();
-        assertEquals(0, phaseTapChanger.getTapPosition());
+        assertEquals(0, phaseTapChanger.getTapPosition().orElseThrow(AssertionError::new));
```

Comment:
I known that there is many places in powsybl where we use AssertionError fir this kind of purpose but this is a bad practice. java.lang.Error are not easy (and very rarely) catched and could be mistaken with VM internal java.lang.Error so for error that could come up we shoud use a standard runtime exception. (for instance it is not the same a a switch case default that cannot be thrown because of cases have been covered)